conda/testing/helpers.py

```diff
@@ -32,6 +32,7 @@
     "CONDA_TEST_DATA_DIR", abspath(join(dirname(__file__), "..", "..", "tests", "data"))
 )
 CHANNEL_DIR = abspath(join(TEST_DATA_DIR, "conda_format_repo"))
```

Comment:
Worth renaming to have a V1 suffix?

Reply:
Added it as an alias and renamed elsewhere (in case it was being used elsewhere)